README.md

```diff
@@ -111,13 +112,34 @@ Write a conditional statement that...
 ### Your solution here:
 8.  Write your javascript solution below
 ```js
-// your answer here
+let num= 3;
+if(num%3===0 && num%5===0){
+    console.log('FizzBuzz');
+}else if(num%5===0){
+    console.log('Buzz');
+
+}else if (num%3===0){
+console.log('Fizz');
+}else{
+console.log(num);
+}
 ```
 
 #### BONUS
 9.  Research a [loop](https://javascript.info/while-for) so that your condition runs on every number from 0 to 100
 ```js
-// your answer here
+for(i=0,i<=100,i++){
+  if(i%3===0 && i%5===0){
+    console.log('FizzBuzz');
+}else if(i%5===0){
+    console.log('Buzz');
+
+}else if (i%3===0){
+console.log('Fizz');
+}else{
+console.log(num);
+}  
+}
 ```
 10.  Research a [function](https://javascript.info/function-basics) so that your condition runs on every number from 0 to whatever number is passed into the function
```

Comment:
`const fizzBuzz = function (number){
    for (let i = 0; i <= number; i++){
        if (i % 3 == 0 &&  i % 5 == 0 ) {
            console.log("FizzBuzz")

        } else if (i % 3 == 0) {
            console.log("Fizz")  

        } else if (i % 5 == 0){
            console.log("Buzz");
        }
    }
}`